predicthq/endpoints/v1/events/schemas.py

```diff
@@ -61,6 +64,8 @@ class Event(Model):
     class Options:
         serialize_when_none = True
 
+    brand_safe = BooleanType()
```

Comment:
I think that `brand_safe` is a paid feature too. Can you double check?

Reply:
Good point. It turns out "paid feature" isn't that well defined. So I've classified all event fields and filters which are plan/subscription-based: 14e03fa